src/api/apollo/generated/graphql.tsx

```diff
@@ -10,157 +10,43 @@ export type Scalars = {
   Boolean: boolean;
   Int: number;
   Float: number;
-  UUID: any;
   Version: any;
   Address: any;
+  UUID: any;
```

Comment:
Do we need to move it down? :)

Reply:
Thanks for reviewing all of this Max. 

In 'graphql.tsx', any changes you see are the result of me having run `npm run graphql:codegen`. The tool reorganizes the generated code. I don't believe I actually needed to run the tool tho, so if you prefer I can revert changes to this file. Or, I can just leave it as-is knowing that the tool is reorganizing the code. Please let me know which is preferrable.